trunk/SUAVE/Analyses/Mission/Segments/Conditions/Aerodynamics.py

```diff
@@ -138,11 +138,10 @@ def __defaults__(self):
         self.propulsion.battery_pack_temperature             = ones_1col * 0
         self.propulsion.battery_cell_temperature             = ones_1col * 0 
         self.propulsion.battery_charge_throughput            = ones_1col * 0    
-        self.propulsion.battery_age_in_days                  = 0.
+        self.propulsion.battery_age                          = 0
```

Comment:
days might be a better descriptor because SUAVE base units are SI hence this should be store in seconds

Reply:
I will change it to battery_cycle_day because I think battery_day would cause confusion.